bin/mc-send-to-console

```diff
@@ -3,26 +3,24 @@
 : "${CONSOLE_IN_NAMED_PIPE:=/tmp/minecraft-console-in}"
 
 if isFalse "${CREATE_CONSOLE_IN_PIPE:-false}"; then
-  echo "ERROR: console pipe needs to be enabled by setting CREATE_CONSOLE_IN_PIPE to true"
+  error "Console pipe needs to be enabled by setting CREATE_CONSOLE_IN_PIPE to true"
 fi
 
 
 if [ $# = 0 ]; then
-  echo "ERROR: pass console commands as arguments"
+  error "Pass console commands as arguments"
   exit 1
 fi
 
 if [ ! -p "${CONSOLE_IN_NAMED_PIPE}" ]; then
-  echo "ERROR: named pipe ${CONSOLE_IN_NAMED_PIPE} is missing"
+  error "Named pipe ${CONSOLE_IN_NAMED_PIPE} is missing"
   exit 1
 fi
 
+commandSuffix="${CONSOLE_IN_NAMED_PIPE:-/tmp/minecraft-console-in}"
```

Comment:
CONSOLE_IN_NAMED_PIPE is already a variable initialized on line 3. No need to assign again to a variable.